www/scipylib/faq.rst

```diff
@@ -500,13 +499,18 @@ and a scalar:
 Where to get help
 -----------------
 
-You can ask questions with the `Scipy tag on Stackoverflow
+You can ask questions with the `SciPy tag on Stackoverflow
 <http://stackoverflow.com/questions/tagged/scipy>`_, or on the scipy-user
 :doc:`mailing list <mailing-lists>`. Search for an answer first, because someone
 may already have found a solution to your problem, and using that will save
 everyone time.
 
 .. Links
-.. _f2py: http://www.f2py.com/
-.. _Enthought: https://www.enthought.com/
-.. _Matplotlib: http://matplotlib.org/
+.. _f2py: http://www.f2py.com
+.. _Enthought: https://www.enthought.com
+.. _Anaconda: https://www.anaconda.com
+.. _Quansight: https://www.quansight.com
+.. _Matplotlib: http://matplotlib.org
+.. _Bokeh: https://bokeh.pydata.org/en/latest
+.. _Altair: https://altair-viz.github.io
+.. _Plotly: https://plot.ly
```

Comment:
The canonical link for PyPy is https://www.pypy.org, not sure how to link it in to my comment above

Reply:
added, thanks